src/tds/config.c

```diff
@@ -700,7 +700,8 @@ tds_parse_conf_section(const char *option, const char *value, void *param)
 	}
 
 	if (!s || got_error) {
-		login->valid_configuration = 0;
+		if (login)
+			login->valid_configuration = 0;
```

Comment:
`login` is used dozens of times in this function, we would have crash much sooner than here

Reply:
The previous uses are all contingent on specific option values, so a (hypothetical!) call along the lines of
```C
tds_parse_conf_section("unimplemented", value, NULL);
```
could otherwise crash right there, but I don't know why the analyzer singled that one line out.